### PR TITLE
Releasing Azure RM sdk version 0.3.1

### PR DESCRIPTION
**Pull request by vishrutshah**
This release corrects the gemspec files to point to the right Homepage for each azure_mgmt_* gems.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2016.05.23 - Azure ARM version 0.3.1
+* Regenerated ARM sdk with Autorest [autorest.0.17.0-Nightly20160519](https://www.myget.org/feed/autorest/package/nuget/autorest/0.17.0-Nightly20160522)
+* Updating gemspec files to use the correct [homepage](https://github.com/Azure/azure-sdk-ruby) for each gems
```
Comment (veronicagg) [inline ChangeLog.md]: minor: 'for each gem' (removing the last 's')

Comment (vishrutshah) [inline ChangeLog.md]: Sure corrected

**Comment by veronicagg** [comment]
@vishrutshah Looks good!

**Comment by vishrutshah** [comment]
Thanks for the review !!

**Pull request merged**